doc/Build.md

```diff
@@ -41,10 +41,18 @@ Optional: Install NodeJS from node's apt repository:
 
 ```shell
 curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
-sudo apt-get install -y nodejs
 ```
+...and check which version of node you're getting, with:
+```shell
+apt-cache policy nodejs
+```
+...and substitute appropriately for `6.14.1-1nodesource1` in the following command:
+```shell
+sudo apt-get install -y nodejs=6.14.1-1nodesource1
```

Comment:
Wouldn't this line work as is to install the requested version without the checks above?